src/image_editing.rs

```diff
@@ -384,7 +384,7 @@ impl ImageOperation {
                 }
                 r
             }
-            Self::Blur(val) => ui.slider_styled(val, 0..=20),
+            Self::Blur(val) => ui.slider_styled(val, 0..=254),
```

Comment:
Any reason 254 instead of 255/256? 255 is a more common number found and 256 of course follows bit increments.

Reply:
Yeah, it is a limitation of the filter:
https://docs.rs/libblur/latest/libblur/fn.stack_blur.html

`radius - radius is limited into 2..254`